src/lib/utils/constants.ts

```diff
@@ -2,3 +2,6 @@ import { pathToFileURL } from 'node:url';
 import { getRootData } from '@sapphire/pieces';
 
 export const rootURL = pathToFileURL(`${getRootData().root}/`);
+
+// Might be changed in the future after more testing.
+export const kGuessThreshold = 0.9 as const;
```

Comment:
A more descriptive comment should be given since it's taken kinda out of context (although it kinda was all along, mb). Also consider renaming it? Idrc but technically it's not just for guessing anymore.
 
```suggestion
/*
 * The threshold to use with jaro-winkler when validating guesses and cleaning song names.
 * Might be changed in the future after more testing.
 */
export const kGuessThreshold = 0.9 as const;
```